Cloudinary/Shared/Coudinary.NetCoreShared/Url.cs

```diff
@@ -211,7 +324,7 @@ public string BuildSpriteCss(string source)
         /// </summary>
         /// <param name="source">A Cloudinary public ID or file name or a reference to a resource.</param>
         /// <param name="keyValuePairs">Array of strings in form of "key=value".</param>
-        public IHtmlContent BuildImageTag(string source, params string[] keyValuePairs)
+        public string BuildImageTag(string source, params string[] keyValuePairs)
```

Comment:
Absolutely not. It was checked and changed in places where IHTMLString was used. Don`t worry about it.